src/ucp/proto/rndv.c

```diff
@@ -986,7 +987,8 @@ static void ucp_rndv_do_rkey_ptr(ucp_request_t *rndv_req, ucp_request_t *rreq,
         /* We should be able to find a lane, because ucp_rndv_is_rkey_ptr()
          * already checked that (rkey->md_map & ep_config->rkey_ptr_dst_mds) != 0
          */
-        ucs_fatal("failed to find a lane to access remote memory domains 0x%lx",
+        ucs_fatal("failed to find a lane to access remote memory domains "
+                  "0x%"PRIx64,
```

Comment:
unite rkey->md_map and PRIx64 to same line

Reply:
done